src/common/Select.jsx

```diff
@@ -28,18 +29,18 @@ const Select = props => (
         ))
       }
     </select>
-    </div>
+  </div>
 );
 /**
-    * @name propTypes
-    * @type {PropType}
-    * @param {Object} propTypes - React PropTypes
-    * @property {String} name - The name of the select element
-    *@property {String} title - The title of the label
-    *@property {String} option - string to be shown in the dropdown
-    *@property {String} selectedOption - The initial selected value
-    *@property {placeholde} selectedOption - plalace holder text
-  */
+  * @name propTypes
```

Comment:
The JsDoc is not aligned properly from L35-L42. It should be 
```
/**
 * @summary description
 */
```
Also other instances in other files.